common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
```

Comment:
Soz came from hastily prepared thrasher, am switching it all to live under the `this-is-the-nhs` block.